torch/fx/node.py

```diff
@@ -51,6 +51,46 @@ def __init__(self, graph: 'Graph', name: str, op: str, target: Target,
         # does not produce a value, it's more of a notation. Thus, this value
         # describes the type of args[0] in the `return` node.
         self.type : Optional[Any] = type
+        self._prev = self
+        self._next = self
+        self._erased = False
+
+    @property
+    def next(self) -> 'Node':
+        return self._next
+
+    @property
+    def prev(self) -> 'Node':
+        return self._prev
+
+    def prepend(self, x: 'Node'):
+        """Insert x before this node in the list of nodes in the graph.
+           Before: p -> self
+                   bx -> x -> ax
+           After:  p -> x -> self
+                   bx -> ax
```

Comment:
It took me a second to figure out what the `bx` `ax` business meant. Maybe drop a line of prose at 67 saying "If `x` is already part of a node list, it is removed from that node list." or something?